client/components/status/status.js

```diff
@@ -623,14 +773,31 @@ export default class Status extends React.Component {
           className={session.stop_time === null ? "active-session" : ""}
         >
           <th>{session_info.header.download}:</th>
-          <td>{this.getMB(session.output_octets)}</td>
+          <td>
+            {bytes(session.output_octets, {
+              decimalPlaces: 0,
+              unitSeparator: " ",
+              unit: "MB",
+            })}
+          </td>
         </tr>
         <tr
           key={`${session.session_id}upload`}
           className={session.stop_time === null ? "active-session" : ""}
         >
           <th>{session_info.header.upload}:</th>
-          <td>{this.getMB(session.input_octets)}</td>
+          <td>
+            {bytes(session.input_octets, {
+              decimalPlaces: 0,
+              unitSeparator: " ",
+              unit: "MB",
+            })}
+            {bytes(session.output_octets, {
+              decimalPlaces: 0,
+              unitSeparator: " ",
+              unit: "MB",
+            })}
+          </td>
```

Comment:
I think there's a bug here, this is the result on a narrow screen:

![Screenshot from 2024-01-12 16-27-12](https://github.com/openwisp/openwisp-wifi-login-pages/assets/841044/6b80ceab-3553-4dc9-8ae3-74435a42b3e2)

I think `session.output_octets` must be removed here.